### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by SaiTejaKondapalli**
@rehana-ms @LaxmiNarayanaMurthyVemuri  Can you please verify and suggest changes if any

### CODE REVIEW

File: Project 1/import.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+b = open("books.csv")
+books = csv.reader(b)
+i = 0
+for ISBN, title, author, year in books:
+    if i != 0:
+        book = Book(isbn=ISBN, title=title, author=author, year=year)
+        print(book.title)
+        db.add(book)
+    i += 1
+db.commit()
```
Comment (rehana-ms) [inline Project 1/import.py:19]: How many records are you able to load into your table.

File: Project 1/application.py [hunk 2]
```diff
@@ -24,18 +24,30 @@
...
+            name = "Not registered user !!! Please register here"
+        return render_template("index.html",name=name)
     elif request.method == "POST":
         session["data"] = []
         name = request.form['name']
+        session["data"].append(name)
         email = request.form['email']
+        session["data"].append(email)
         pswd1 = request.form['pswd']
         pswdhash1 = hashlib.md5(pswd1.encode()).hexdigest()
```
Comment (rehana-ms) [inline Project 1/application.py:49]: Good that  you are using hashing the password.But while verifying password,where are you doing it or  how are you doing it.

File: Project 1/import.py (hunk 1 above)
Comment (SaiTejaKondapalli) [reply Project 1/import.py:19]: 5000 records are loaded into the table mam. I have checked in the Adminer.

File: Project 1/application.py (hunk 2 above)
Comment (SaiTejaKondapalli) [reply Project 1/application.py:49]: While verifying passwords, I have used the condition for non-hashed passwords. I need to change the condition to check whether the hashed passwords are equal or not mam.

Comment (rehana-ms) [reply Project 1/application.py:49]: Ok,please do and update.Will check.

Comment (SaiTejaKondapalli) [reply Project 1/application.py:49]: I have updated the condition and checked by filling the details. It is working mam.

Comment (rehana-ms) [reply Project 1/application.py:50]: You are checking password and re-password fields.But are you doing while login also.

Comment (SaiTejaKondapalli) [reply Project 1/application.py:50]: Yes mam I am checking while login also. In the /auth route I am checking the condition whether the hashed password taken from the login form is equal to the hashed password of user in the table.

Comment (rehana-ms) [reply Project 1/application.py:50]: I think you used hash function during login also,but you will not get same hash value even though you enter same password. Check by printing it after hashing,you will see the difference.Instead use verify method. Pls refer to this link https://pythonprogramming.net/password-hashing-flask-tutorial/

Comment (rehana-ms) [reply Project 1/application.py:50]: Otherwise everything seems fine.

Comment (SaiTejaKondapalli) [reply Project 1/application.py:50]: Sure mam, I will go through the link and change it.

Comment (SaiTejaKondapalli) [reply Project 1/application.py]: Modified the code, used the verify method while hashing and checking passwords.

Comment (rehana-ms) [reply Project 1/application.py]: :+1:

File: Project 1/import.py (hunk 1 above)
Comment (rehana-ms) [reply Project 1/import.py:19]: :+1:

**Pull request merged**